docroot/themes/custom/mass_theme/templates/content/node--topic-page.html.twig

```diff
@@ -69,11 +69,22 @@
       {% endblock %}
     {% endembed %}
   {% endif %}
+  {# Build the pageHeader variable dynamically #}
+  {% set pageHeader = {
+    "title": node.label,
+    "underline": "TRUE",
+  } %}
+
+  {# Check if 'field_display_short_description' is checked #}
+  {% if node.field_display_short_description.value == '1' %}
+    {% set pageHeader = pageHeader|merge({
+      "description": node.field_topic_lede.value
```

Comment:
@arthurbaghdas  please add a simple test to check this functionality.

Reply:
Added @dstorozhuk